sdk/identity/Azure.Identity/src/DefaultAzureCredential.cs

```diff
@@ -92,7 +92,7 @@ public override async ValueTask<AccessToken> GetTokenAsync(TokenRequestContext r
 
         private async Task<AccessToken> GetTokenAsync(bool isAsync, TokenRequestContext requestContext, CancellationToken cancellationToken)
         {
-            using CredentialDiagnosticScope scope = _pipeline.StartGetTokenScope("Azure.Identity.DefaultAcureCredential.GetToken", requestContext);
+            using CredentialDiagnosticScope scope = _pipeline.StartGetTokenScope("Azure.Identity.DefaultAzureCredential.GetToken", requestContext);
```

Comment:
As an FYI - we've been using `$"{nameof(Azure)}.{nameof(Identity)}.{nameof(DefaultAzureCredential)}.{nameof(GetToken)}"` for some of the newer ones in Storage.  Might be worth a refactor the next time we make a global change to the distributed tracing code.